modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -330,6 +349,16 @@ class Emails {
     session.log(' - password is correct, userId: ${entry.userId})',
         level: LogLevel.debug);
 
+    var migratedAuth = Emails.tryMigrateAuthEntry(
+      password: password,
+      entry: entry,
+    );
+
+    if (migratedAuth != null) {
+      session.log(' - migrated auth', level: LogLevel.debug);
+      await EmailAuth.db.updateRow(session, migratedAuth);
+    }
```

Comment:
Question: Shouldn't we have some fail safes here? This call could fail to update the database and it would be nice to then give a graceful error (well internal server error).